src/Client/ConcordiumClient.cs

```diff
@@ -48,20 +47,29 @@ public sealed class ConcordiumClient : IDisposable
     /// </param>
     /// <param name="timeout">The maximum permitted duration of a call made by this client, in seconds. <c>null</c> allows the call to run indefinitely.</param>
     /// <param name="rawChannelOptions">The options for the channel that is used to communicate with the node.</param>
+    [Obsolete($"Use {nameof(ConcordiumClient)} with overloads which accepts ${nameof(ConcordiumClientOptions)}")]
```

Comment:
It seems that the generated documentation does contain the information of this being obsolete, so I guess we have to add that manually

Reply:
Added it in summary since I wasn’t able to use any tags or attributes which added this.